src/utils/forecast.js

```diff
@@ -12,10 +12,29 @@ const forecast = (lat, lon, callback) => {
                 description: body.current.weather_descriptions[0],
                 temperature: body.current.temperature,
                 feelslike: body.current.feelslike,
-                icon: body.current.weather_icons[0]
+                icon: body.current.weather_icons[0],
+                location: body.location.region
             })
         }
     })
 }
 
-module.exports.forecast = forecast
\ No newline at end of file
+const forecastErrCheck = (error, { description, temperature, feelslike, icon, location }) => {
```

Comment:
sounds more naturally
```suggestion
const checkForecaseError = (error, { description, temperature, feelslike, icon, location }) => {
```

Reply:
Actually, the promise wrapping in useless.
`geocode.geocode` and `forecast` should be async functions without callback passing

Reply:
Removed this method altogether.
Please see current geocode, forecast implementation.